examples/exampletvOS/ios/Podfile

```diff
@@ -9,7 +9,7 @@ production = ENV["PRODUCTION"] == "1"
 
 target 'exampletvOS' do
 
-  platform :ios, min_ios_version_supported
```

Comment:
@KrzysztofMoch I think we need to document this point also ! 
Would that fix be better ? https://github.com/TheWidlarzGroup/react-native-video/issues/3737#issuecomment-2096624471

Reply:
Yes, I am also thinking about adding this info to README (that min ios version is 13) and to upgrade helper